README.md

```diff
@@ -60,9 +63,39 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+yum -y install epel-release
+yum -y install \
+    libibverbs \
+    NetworkManager \
+    NetworkManager-libnm \
+    NetworkManager-ovs \
+    openvswitch \
+    python2-pyyaml \
```

Comment:
`dbus-python` should also be installed on CentOS/RHEL 7.

Reply:
I think this is something you are adding through a different PR.
How come it works as is at the moment?

Reply:
Currently it will not use the checkpoint feature if it is not installed but it would already use it if it is there. On RHEL it is typically installed for subscription manager etc. Not sure about CentOS.

Reply:
Done